service/src/main/java/uk/gov/hmcts/reform/fpl/model/Address.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Address {
```

Comment:
You did not really have to implement it, ignoring unknown properties in Jackson would be simplest but since you already done it let's leave it for the future.